src/libs/KeyboardAvoidingView/index.android.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {KeyboardAvoidingView as KeyboardAvoidingViewComponent} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+
+function KeyboardAvoidingView({children}) {
+    return (
+        <KeyboardAvoidingViewComponent
+            enabled={false}
+            behavior="height"
+            style={[styles.w100, styles.h100]}
+        >
+            {children}
+        </KeyboardAvoidingViewComponent>
+    );
+}
+KeyboardAvoidingView.propTypes = {
+    children: PropTypes.node,
+};
+KeyboardAvoidingView.defaultProps = {
+    children: null,
+};
```

Comment:
To match our [style guide](https://github.com/Expensify/Expensify.cash/blob/main/STYLE.md) and stay consistent:

> All propTypes and defaultProps must be defined at the top of the file in variables called propTypes and defaultProps. These variables should then be assigned to the component at the bottom of the file.